packages/zoe/README.md

```diff
@@ -51,3 +51,60 @@ agoric follow :published.priceFeed.ATOM-USD_price_feed
 ```
 
  TODO document more of https://github.com/Agoric/documentation/issues/672
+
+## Upgrade
+
+A contract instance can be upgraded to use a new source code bundle in a process that is very similar to
+[upgrading a vat](https://github.com/Agoric/agoric-sdk/blob/master/packages/SwingSet/docs/vat-upgrade.md).
+Most state is discarded, however "durable" collections are retained for use by the replacement version.
+
+The upgrade process is triggered through the "adminFacet" of the instance, and requires specifying the new source bundle. (Note that a "null upgrade" that re-uses the original bundle is valid, and a legitimate approach to deleting accumulated state).
+
+```js
+const results = E(instanceAdminFacet).upgradeContract(newBundleId);
```

Comment:
FWIW, I've been trying to use `bundleID` instead of `bundleId`. It looks like we aren't consistent, but I count 440 `bundleID`s vs only 21 `bundleId`, so maybe consider joining my "the bartender asks for your ID, not your id" camp :).